src/libraries/iKin/src/iKinSlv.cpp

```diff
@@ -438,7 +439,7 @@ void CartesianSolver::getFeedback(const bool wait)
         while (wait && !closing && !ok)
         {
             ok=enc[i]->getEncoders(fbTmp.data());
-            Time::delay(0.00025*getRate()); // wait for 1/4 of thread period
+            Time::yield();
```

Comment:
Warning: I am not sure how much `Time::yield` is equivalent to `Time::delay(...)`.

Reply:
@traversaro I know, but when I used `Time::delay` I was clearly not aware of `Time::yield` 😉

Reply:
Yep, I just wanted to let you know that my approval was actually "I do not know what is going on" relativly to this line.